internal/store/postgres/migrations/20220819074139_add_metadata_keys_table.down.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS metadata_keys;
```

Comment:
can you add new line at end of these sql files? [thats how posix define a file](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_205)